twentytwenty-blocks/style.css

```
@@ -0,0 +1,76 @@
+/*
+Theme Name: Twenty Twenty
+Author: the WordPress team
+Author URI: https://wordpress.org/
+Description: Twenty Twenty, but only using block templates.
+Requires at least: WordPress 4.9.6
+Version: 0.03
+License: GNU General Public License v2 or later
+License URI: LICENSE
+Text Domain: twentytwenty
```

```suggestion
Text Domain: twentytwenty-blocks
```
^ Update to match the theme's folder name.